src/Popup/PopupInsert/PopupActionGroup/index.js

```diff
@@ -23,8 +23,10 @@ class PopupActionGroup extends Component {
   }
 
   render () {
-    const { title, children } = this.props
+    const { title, children, feature } = this.props
     const { right, top } = this.el ? this.el.getBoundingClientRect() : { right: 0, top: 0 }
+    
+    const transformedChildren = React.Children.map(children, c => React.cloneElement(c, { feature }))
```

Comment:
@ntatko can you always `.map` children? even is there's only one? I remember React being dumb about this

Reply:
Yeah, this is the safe way. `React.Children.map` !== `children.map` - you can pass only one node, and React knows what to do.